src/CarbonAware.Aggregators/src/CarbonAware/CarbonAwareAggregator.cs

```diff
@@ -86,7 +88,7 @@ public async Task<EmissionsForecast> GetForecastDataAsync(IDictionary props)
         {
             ValidateInput(props);
 
-            var locations = (IEnumerable<Location>) props[CarbonAwareConstants.Locations]!;
+            var locations = (IEnumerable<Location>)props[CarbonAwareConstants.Locations]!;
```

Comment:
🐑 This should be `GetLocationOrThrow` no? (I don't think you actually wrote this but the whitespace change drew my attention

Reply:
I think it was intentionally done this way, since `ValidateInput` does all the "locations" validating.